dirsrvtests/tests/suites/vlv/regression_test.py

```diff
@@ -103,53 +112,53 @@ def check_vlv_search(conn, **kwargs):
 
     for i, (dn, entry) in enumerate(result, start=imin):
         assert i <= imax
-        expected_dn = f'uid=testuser{i},ou=People,dc=example,dc=com'
-        log.debug(f'found {dn} expected {expected_dn}')
-        assert dn.lower() == expected_dn.lower()
+        expected_rdn = f'uid=testuser{i},'
+        log.debug(f'found {dn} should starts with {expected_rdn}')
+        assert dn.lower().startswith(expected_rdn)
 
 
 def add_user(inst, users, uid):
-        user_properties = {
-            'uid': f'testuser{uid}',
-            'cn': f'testuser{uid}',
-            'sn': 'user',
-            'uidNumber': str(uid),
-            'gidNumber': str(uid),
-            'homeDirectory': f'/home/testuser{uid}'
-        }
-        users.create(properties=user_properties)
-
-
-def add_users(inst, users_num):
-    users = UserAccounts(inst, DEFAULT_SUFFIX)
+    user_properties = {
+        'uid': f'testuser{uid}',
+        'cn': f'testuser{uid}',
+        'sn': 'user',
+        'uidNumber': str(uid),
+        'gidNumber': str(uid),
+        'homeDirectory': f'/home/testuser{uid}'
+    }
+    users.create(properties=user_properties)
+
+
+def add_users(inst, users_num, suffix=DEFAULT_SUFFIX):
+    users = UserAccounts(inst, suffix)
     log.info(f'Adding {users_num} users')
     for i in range(0, users_num):
         uid = STARTING_UID_INDEX + i
         add_user(inst, users, uid)
 
 
-def create_vlv_search_and_index(inst, hyphen='', basedn=DEFAULT_SUFFIX):
+def create_vlv_search_and_index(inst, basedn=DEFAULT_SUFFIX, bename='userRoot', scope=2, prefix="vlv"):
```

Comment:
For better readability, I think we should use `ldap.LDAP_SCOPE_SUBTREE` for the scope here, instead of magic number.

Reply:
OK, will use ldap.SCOPE_SUBTREE instead of 2